README.md

```diff
@@ -31,10 +31,9 @@ settings under Edit > Project Settings > Quality:
 ## Requirements
 
 * Windows / OSX
-* Unity version: 2018.3 (2018.3.4f1 and up) and 2019.1 (2019.1.0b2 and up)
-See ["2017 users" section](#unity-2017-users) if you want to use the plugin in Unity 2017.4
-* To build a standalone, Api Compatibility Level needs to be set to .NET 4.x
-   In Edit > Project Settings > Player :
+* Unity version: 2019.4 and up
+* Api profile .NET 4.x is no longer required but still provides better performances than .NET 2   
```

Comment:
Is `profile` the correct word here? Maybe we could say `Api Compatibility Level` to mirror what is in the interface?

Reply:
Yeah profile is the term used in the doc, but I'll change that

Reply:
Done